src/component/mxgraph/StyleConfigurator.ts

```diff
@@ -174,32 +191,19 @@ export default class StyleConfigurator {
     style[this.mxConstants.STYLE_FILLCOLOR] = 'White';
     style[this.mxConstants.STYLE_FONTCOLOR] = 'Black';
     style[this.mxConstants.STYLE_LABEL_BACKGROUNDCOLOR] = 'none';
-    style[this.mxConstants.STYLE_VERTICAL_LABEL_POSITION] = 'top';
+    style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'bottom';
   }
 
   private configureSequenceFlowsStyle(): void {
-    this.configureNormalSequenceFlowStyle();
-    this.configureDefaultSequenceFlowStyle();
-    this.configureConditionalSequenceFlowStyle();
-  }
-
-  private configureNormalSequenceFlowStyle(): void {
-    const style = this.cloneDefaultEdgeStyle();
-    this.graph.getStylesheet().putCellStyle(SequenceFlowKind.NORMAL, style);
-  }
-
-  private configureDefaultSequenceFlowStyle(): void {
-    const style = this.cloneDefaultEdgeStyle();
-    style[this.mxConstants.STYLE_STARTARROW] = MarkerConstant.ARROW_DASH;
-    this.graph.getStylesheet().putCellStyle(SequenceFlowKind.DEFAULT, style);
-  }
-
-  private configureConditionalSequenceFlowStyle(): void {
-    const style = this.cloneDefaultEdgeStyle();
-    style[this.mxConstants.STYLE_STARTARROW] = this.mxConstants.ARROW_DIAMOND_THIN;
-    style[this.mxConstants.STYLE_STARTSIZE] = 20;
-    style[this.mxConstants.STYLE_STROKECOLOR] = 'Chartreuse';
-    style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'bottom';
-    this.graph.getStylesheet().putCellStyle(SequenceFlowKind.CONDITIONAL, style);
+    Object.values(SequenceFlowKind).forEach(kind => {
+      const style = this.cloneDefaultEdgeStyle();
+      const updateEdgeStyle =
+        this.specificEdgeStyles.get(kind) ||
+        (() => {
+          // Do nothing
+        });
```

Comment:
:question: probably clear enough without the explicit comment (to be tested, suggestion may not follow the right linting)
```suggestion
        this.specificEdgeStyles.get(kind) || (() => {});
```

Reply:
I added the comment, because I had a error in the IDE

Reply:
ok, so forget about my comment :smiley_cat: